openproblems/tasks/dimensionality_reduction/methods/densmap.py

```diff
@@ -0,0 +1,41 @@
+from ....tools.decorators import method
+from ....tools.utils import check_version
+from .preprocessing import preprocess_logCPM_1kHVG
+
+
+@method(
+    method_name="densMAP (logCPM, 1kHVG)",
+    paper_name="Assessing single-cell transcriptomic variability through"
+    " density-preserving data visualization",
+    paper_url="https://www.nature.com/articles/s41587-020-00801-7",
+    paper_year=2021,
+    code_url="https://github.com/lmcinnes/umap",
+    code_version=check_version("umap-learn"),
+    image="openproblems-python-extras",
+)
+def densmap_logCPM_1kHVG(adata):
```

Comment:
per #330 these methods will all have to have a kwarg `test: bool=False`

Reply:
Added the argument to all the methods but not currently used.

Reply:
No worries, I don't think it's necessary to use it in all methods